packages/playground/app/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "just-the-docs",
+  "version": "0.2.5",
+  "description": "A modern Jekyll theme for documentation",
+  "repository": "pmarsceill/just-the-docs",
+  "license": "MIT",
+  "bugs": "https://github.com/pmarsceill/just-the-docs/issues",
+  "devDependencies": {
+    "stylelint": "^10.0.1",
+    "stylelint-config-primer": "^7.0.0",
+    "stylelint-selector-no-utility": "^4.0.0"
+  },
+  "dependencies": {
+    "@primer/css": "^12.3.1",
+    "npm-run-all": "^4.1.5",
```

Comment:
npm-run-all, replace-in-file and rimraf are most probably dev deps.

Not sure about the @primer/css. By scrolling the PR I did not find its usage but if we do not use it in the runtime just move it to the devDeps.